AppCenter/AppCenterTests/MSUtilityTests.m

```diff
@@ -897,6 +897,28 @@ - (void)testObfuscateRedirectUri {
   XCTAssertFalse([obfuscatedString rangeOfString:kMSRedirectUriObfuscatedTemplate].location == NSNotFound);
 }
 
+- (void)testDispatchObjectMacro {
+
+  // If
+  NSMutableArray *array = [NSMutableArray new];
+
+  // When
+  MS_DISPATCH_SELECTOR((void (*)(id, SEL, id)), array, addObject:, @"test");
+
+  // Then
+  XCTAssertEqual([array count], 1);
+  XCTAssertEqual([array firstObject], @"test");
+}
+
+- (void)testDispatchObjectMacroWithNil {
+  typedef void (^block)(NSString *, NSString *);
+  MS_DISPATCH_SELECTOR((void (*)(id, SEL, NSString *, NSString *, block)), self, methodWithArgs:secondArg:completionHandler:, nil, @"test",
+                       ^(NSString *firstArg, NSString *secondArg) {
```

Comment:
🤔 I think the test will succeed if the completion is not called. OK now I think we should use a mock.

Reply:
An expectation also work yes.